01-bubble-sort/src/components/Button/Button.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import style from './Button.module.scss';
+
+interface IProps {
+  name: string;
+  fn(): void;

```

Comment:
I would say it's really bad naming, it's unclear what is it at all. `onClick` is more suitable.